cmaes/cma.py

```diff
@@ -281,6 +283,7 @@ def tell(self, solutions: List[Tuple[np.ndarray, float]]) -> None:
         self._sigma *= np.exp(
             (self._c_sigma / self._d_sigma) * (norm_p_sigma / self._chi_n - 1)
         )
+        self._sigma = min(self._sigma, sys.float_info.max)
```

Comment:
Since the `self._sigma` can be used as `self._sigma * SOMETHING`, we need to give the value of `self._sigma` some leeway. That `SOMETHING` is a sample from the standard Gaussian distribution or its linear combination of coefficients sum of 1, so if we set `sys.float_info.max / 5`, we can bound the error of overflow by `P(X \in 5 \sigma confidence interval) ~ 0.000057%`.

```suggestion
        self._sigma = min(self._sigma, sys.float_info.max / 5)
```

Reply:
Thank you for pointing it out. Sounds reasonable.